src/mrinufft/trajectories/inits/random_walk.py

```diff
@@ -88,8 +95,14 @@ def _initialize_ND_random_walk(
 
 
 def initialize_2D_random_walk(
-    Nc, Ns, density, *, diagonals=True, pseudo_random=True, **sampling_kwargs
-):
+    Nc: int,
+    Ns: int,
+    density: np.typing.NDArray,
+    *,
+    diagonals: bool = True,
+    pseudo_random: bool = True,
+    **sampling_kwargs: Literal | bool,
+) -> np.ndarray:
```

Comment:
should be `NDArray`

Reply:
Yes I missed a lot of them for some reason, thx